ckanext/videoview/theme/templates/video_view.html

```diff
@@ -0,0 +1,5 @@
+<video style="margin:auto; max-height:100%; display:block" controls>
+  <source src="{{ resource_view.get('video_url') or resource.get('url') }}" type="{{ resource_view.get('format') }}">
```

Comment:
You need to use resource format here, not the view one:

```suggestion
  <source src="{{ resource_view.get('video_url') or resource.get('url') }}" type="{{ resource.get('format') }}">
```